test/index.test.js

```diff
@@ -148,6 +154,27 @@ describe('index.js', () => {
       });
       expect(message.msg).to.be.eql('hello');
     });
+
+    describe('sensitive data', () => {
+      it('must replace sensitive data with __SENSITIVE_DATA__ in logs', () => {
+        logger.info({
+          password: 'My personal password',
+          headers: {
+            'accept-language': 'fr-FR', // unchanged
+            authorization: 'Bearer token'
+          },
+          req: {
+            token: 'My personal token'
+          }
+        }, 'Logging amazingly sensitive data!');
+        const message = logs.shift();
+
+        expect(message.password).to.equal('__SENSITIVE_DATA__');
+        expect(message.headers['accept-language']).to.equal('fr-FR');
+        expect(message.headers.authorization).to.equal('__SENSITIVE_DATA__');
+        expect(message.req.token).to.equal('__SENSITIVE_DATA__');
```

Comment:
Just asking why haven't you considered a `deepEqual` here, are there other fields you can't control?

Reply:
Yes because the final result is controlled by `lodash`.